### PR TITLE
feat: added potential initiatives for 2023

### PR DESCRIPTION
**Pull request by UlisesGascon**
This is an initial proposal, as agreed on [last meeting](https://github.com/nodejs/security-wg/blob/main/meetings/2022-12-08.md?plain=1#L28)

Please let me know if there are missing initiatives or champions in the list.

Related to #846

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,21 @@ the [Node.js TSC][].
...
 The program is managed through the HackerOne platform at [https://hackerone.com/nodejs](https://hackerone.com/nodejs) with further details.
 
+## Current Initiatives
+
+We are currently defining the Initiatives for 2023, [feel free to participate](https://github.com/nodejs/security-wg/issues/846).
+
+| Initiative           | Champion                                         | Status                                   | Links
+|----------------------|--------------------------------------------------|------------------------------------------|-------------------------------------------------
+| Permission Model | [@RafaelGSS](https://github.com/RafaelGSS) |  PR In Review | [PR #44004](https://github.com/nodejs/node/pull/44004)
+| Automate update dependencies | [@facutuesca](https://github.com/facutuesca) | Well defined | [Issue #828](https://github.com/nodejs/security-wg/issues/828)
```
Comment (RafaelGSS) [inline README.md:57]:
```suggestion
| Automate update dependencies | [@facutuesca](https://github.com/facutuesca) | In Progress | [Issue #828](https://github.com/nodejs/security-wg/issues/828)
```

File: README.md [hunk 2]
```diff
@@ -47,6 +47,21 @@ the [Node.js TSC][].
...
+| Flag to emit a warning when vulnerable | [@RafaelGSS](https://github.com/RafaelGSS)  | Started TSC Discussion | _pending_
```
Comment (RafaelGSS) [inline README.md:58]:
```suggestion
| Flag to emit a warning when vulnerable | [@RafaelGSS](https://github.com/RafaelGSS)  | Evaluation | _pending_
```

File: README.md [hunk 3]
```diff
@@ -47,6 +47,21 @@ the [Node.js TSC][].
...
+| Enforce communication with Node.js dependencies | N/A | Under discussion | [Issue #846](https://github.com/nodejs/security-wg/issues/846)
+| Explore using SigStore to sign our releases | N/A | Under discussion | [Issue #846](https://github.com/nodejs/security-wg/issues/846)
+| Figure out how we stand in terms of key supply chain standards/frameworks | N/A | Under discussion | [Issue #846](https://github.com/nodejs/security-wg/issues/846)
+| Explore how the projects stand on the OSSF Scorecard | N/A | Under discussion | [issue #851](https://github.com/nodejs/security-wg/issues/851)
+| Better versioning and management of tools need to build/update dependencies | N/A | Under discussion | [Issue #846](https://github.com/nodejs/security-wg/issues/846)
```
Comment (RafaelGSS) [inline README.md:63]: I think we should wait to include these ones. Once we all agree in the meeting that's doable or creates a discussion to evaluate its possibility, we add it here.

Comment (UlisesGascon) [reply README.md:63]: sounds great!

Comment (RafaelGSS) [reply README.md:63]: Could you remove?

Comment (UlisesGascon) [reply README.md]: Thanks for the ping 🙂. Done in [579728b](https://github.com/nodejs/security-wg/pull/854/commits/579728b50f6baec4786282902adb10e27620331d)

**Review by RafaelGSS — APPROVED** [review]

**Review by fraxken — APPROVED** [review]

**Pull request merged**